proto/topsql_agent.proto

```diff
@@ -13,8 +13,8 @@ option (gogoproto.unmarshaler_all) = true;
 
 // TopSQLAgent is the persistent agent service for TopSQL records
 service TopSQLAgent {
-    // ReportCPUTimeRecords is called periodically (e.g. per minute) to save the in-memory TopSQL records
-    rpc ReportCPUTimeRecords(stream CPUTimeRecord) returns (EmptyResponse) {}
```

Comment:
I think it's a bad practice that we rename the attribute directly. 😢
@breeswish

Reply:
Yes I agree. This is a breaking change and should be performed with careful review.